src/lib/components/CopyCode/styles.scss

```diff
@@ -0,0 +1,38 @@
+@import "../../../styles/settings/colors";
+@import '../../../styles/settings/global';
+
+.w-copy-code-button {
+  background: none;
+  background-image: url("../../../images/icons/file_copy.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+  border: none;
+  border-radius: $GLOBAL_ROUNDED;
+  float: right;
+  height: 48px;
+  padding: 0;
+  width: 48px;
+  z-index: 100;
```

Comment:
Is this needed?

Reply:
Are we referring to the z-index? Probably not, I think I was tinkering with how I would position the button and it lingered.